src/Pages/SingleProduct/SingleProduct.css

```diff
@@ -0,0 +1,76 @@
+.single-product-main{
+  display: flex;
```

Comment:
Just a suggestion to use a grid layout to make your layout look better in desktop screens